### DISCUSSION TITLE
Criando seu primeiro App com Flutter - aula  Material App

### DISCUSSION THREAD

**Discussion by marcosfonceka1711** [balta-io/forum#345] [Tira Dúvidas] [answered]
Estou seguindo a aula, fiz a instalação correta do Flutter mas não consigo entender onde está dando esse erro do Title. Se alguém puder me explicar, agradeço.
![Screenshot_1](https://github.com/balta-io/forum/assets/10660429/5ca77845-334f-4a9a-bf5d-96116fa6fd23)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @marcosfonceka1711, como vai?
Está usando as mesmas versões do curso? Em adicional, chegou a comparar o código com o código do curso para ver se tem alguma inconsistência?

**Reply by marcosfonceka1711** [reply]
Oi professor, prazer em falar com o senhor.
Na verdade não.
Estou utilizando a versão mais recente do Flutter e por consequência disto, existem pequenas modificações na escrita da linguagem.
Mas eu consegui chegar no mesmo resultado que o senhor.

Muito obrigado por ter me respondido.
Tenha um ótimo dia.

![flutter_correto](https://github.com/balta-io/forum/assets/10660429/a3db5d88-f09f-462a-8a5d-99a66e03aa23)

**Reply by andrebaltieri** [reply]
Bom dia, @marcosfonceka1711 o prazer é meu, meu caro!
Show demais!!! 🚀
